pkg/reconciler/taskrun/taskrun.go

```diff
@@ -442,7 +442,7 @@ func (c *Reconciler) updateStatusLabelsAndAnnotations(tr, original *v1beta1.Task
 		go func(metrics *Recorder) {
 			err := metrics.RunningTaskRuns(c.taskRunLister)
 			if err != nil {
-				c.Logger.Warnf("Failed to log the metrics : %v", err)
+				// c.Logger.Warnf("Failed to log the metrics : %v", err)
```

Comment:
I "guess" the log should still be there ? :thinking:

Reply:
:D oops, that was me trying to reduce noise while debugging unit tests.